src/Core/src/Platform/Windows/Styles/Resources.xaml

```diff
@@ -0,0 +1,359 @@
+                                Margin="{TemplateBinding Padding}"
+                                HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
+                                VerticalAlignment="Top" />
+					</Border>
+				</ControlTemplate>
+			</Setter.Value>
+		</Setter>
+	</Style>
+
+	<Style x:Key="JumpListGrid" TargetType="GridView">
+		<Setter Property="ItemTemplate">
+			<Setter.Value>
+				<DataTemplate>
+					<Border Background="{ThemeResource SystemColorControlAccentBrush}" Padding="5" Margin="3" MinHeight="80" MinWidth="80">
+						<TextBlock Text="{Binding}" Style="{ThemeResource SubheaderTextBlockStyle}" VerticalAlignment="Bottom" />
+					</Border>
+				</DataTemplate>
+			</Setter.Value>
+		</Setter>
+	</Style>
+
+	<Style TargetType="Button" x:Key="PaneButton">
+		<Setter Property="FontFamily" Value="{ThemeResource SymbolThemeFontFamily}" />
+		<Setter Property="FontSize" Value="20" />
+		<Setter Property="Height" Value="48" />
+		<Setter Property="Width" Value="48" />
+		<Setter Property="Content" Value="" />
+	</Style>
+
+    <!--<DataTemplate x:Key="CellTemplate">
+		<uwp:CellControl HorizontalContentAlignment="Stretch" Height="{Binding Cell.RenderHeight,RelativeSource={RelativeSource Mode=Self},Converter={StaticResource HeightConverter}}" />
+	</DataTemplate>-->
+
+	<DataTemplate x:Key="TableRoot">
+		<TextBlock Margin="0,20,0,0" Text="{Binding Title,Converter={StaticResource LowerConverter}}" Style="{ThemeResource TitleTextBlockStyle}" Visibility="{Binding Text,RelativeSource={RelativeSource Mode=Self},Converter={StaticResource CollapseWhenEmpty}}" />
+	</DataTemplate>
+
+	<DataTemplate x:Key="TableSection">
+		<TextBlock Margin="0,20,0,0" Text="{Binding Title,Converter={StaticResource LowerConverter}}" Style="{ThemeResource SubtitleTextBlockStyle}" Visibility="{Binding Text,RelativeSource={RelativeSource Mode=Self},Converter={StaticResource CollapseWhenEmpty}}" Foreground="{Binding TextColor,Converter={StaticResource ColorConverter},ConverterParameter=DefaultTextForegroundThemeBrush}" />
+	</DataTemplate>
+	
+	<!--<uwp:ListViewGroupStyleSelector x:Key="ListViewGroupSelector" />-->
+
+	<DataTemplate x:Key="ListViewGroupTemplate">
+		<TextBlock Text="{Binding}" Style="{ThemeResource BaseTextBlockStyle}" />
+	</DataTemplate>
+	
+	<!--<GroupStyle x:Key="ListViewGroup" HidesIfEmpty="False">
+		<GroupStyle.HeaderContainerStyle>
+			<Style TargetType="ListViewHeaderItem">
+				<Setter Property="Template">
+					<Setter.Value>
+						<ControlTemplate>
+							<uwp:ListGroupHeaderPresenter />
+						</ControlTemplate>
+					</Setter.Value>
+				</Setter>
+			</Style>
+		</GroupStyle.HeaderContainerStyle>
+		<GroupStyle.HeaderTemplate>
+			<DataTemplate>
+                <uwp:CellControl IsGroupHeader="true" HorizontalContentAlignment="Stretch" />
+			</DataTemplate>
+		</GroupStyle.HeaderTemplate>
+	</GroupStyle>-->
+	
+	<DataTemplate x:Key="ViewCell">
```

Comment:
It seems that some styles and templates here will not be used in .NET MAUI, should we gradually add what we are adding to this resource dictionary?

Reply:
Good catch!!!

I just removed everything that's not used so we can take it piece by piece